src/Identifier/OrmIdentifier.php

```diff
@@ -46,10 +48,30 @@ class OrmIdentifier extends AbstractIdentifier
         ],
         'userModel' => 'Users',
         'finder' => 'all',
-        'passwordHasher' => DefaultPasswordHasher::class
+        'passwordHasher' => null
     ];
 
     /**
+     * Return password hasher object.
+     *
+     * @return \Authentication\PasswordHasher\PasswordHasherInterface Password hasher instance.
+     */
+    public function getPasswordHasher()
+    {
+        if ($this->_passwordHasher === null) {
+            $passwordHasher = $this->getConfig('passwordHasher');
+            if ($passwordHasher !== null) {
+                $passwordHasher = PasswordHasherFactory::build($passwordHasher);
+            } else {
+                $passwordHasher = $this->traitGetPasswordHasher();
```

Comment:
Thats one of the reasons I really liked the underscore

    _getPasswordHasher()

;)

But well, thhis way is OK as well I guess.